### DISCUSSION TITLE
How to get a list of partition keys in a `DynamicPartitionsDefinition`

### DISCUSSION THREAD

**Discussion by sryza** [dagster-io/dagster#14715] [Q&A] [answered]
E.g.:
- I wanted to use the get_partition_keys method in my sensor to reconcile the existing partitions with the actual partitions.

**Comment by sryza** [comment, marked as answer]
First, get your hands on an instance: https://github.com/dagster-io/dagster/discussions/7427

```python
instance = ...
```

Then:
```python
dynamic_partitions_def = DynamicPartitionsDefinition(name="something")

partition_keys = dynamic_partitions_def.get_partition_keys(dynamic_partitions_store=instance)
```

**Comment by dsgibbons** [comment]
Are there any plans for viewing dynamic partition definitions and their keys directly in the web UI? It would be nice to have a tab alongside Sensors, Jobs etc., that shows the current partition definitions and their keys.
